lib/evm_database.rb

```diff
@@ -95,4 +97,91 @@ def self.ping(connection)
     query = "SELECT 1"
     Benchmark.realtime { 10.times { connection.select_value(query) } } / 10 * 1000
   end
+
+  # Determines if the schema currently being used is the same as the one we expect
+  #
+  # @param connection Check the database at this connection against the local file
+  # @return nil if the schemas match, an error message otherwise
+  def self.check_schema(connection = ActiveRecord::Base.connection)
+    ret = check_schema_tables(connection)
+    return ret if ret
+
+    ret = check_schema_columns(connection)
```

Comment:
Even better, the whole methods can just be 

`check_schema_tables(connection) || check_schema_columns(connection)`